.travis.yml

```diff
@@ -73,7 +72,8 @@ script:
 - ./tools/lint.py
 - ./tools/test_format.py
 - ./tools/build.py -C target/release
-- DENO_BUILD_MODE=release ./tools/test.py
+- cargo build -p test_plugin --release --locked
```

Comment:
I don't like that this breaks my usual `./tools/build.py && ./tools/test.py` 

Let me see if I can whip up a BUILD.gn to do this...

Reply:
I've added the BUILD.gn file... it's a bit hacky. PTAL @afinch7